docs/sources/reference/stdlib/encoding.md

```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/stdlib/encoding/
+description: Learn about encoding functions
+aliases:
+  - ./base64_decode/ # /docs/alloy/latest/reference/stdlib/base64_decode/
+  - ./json_decode/ # /docs/alloy/latest/reference/stdlib/json_decode/
+  - ./yaml_decode/ # /docs/alloy/latest/reference/stdlib/yaml_decode/
+menuTitle: encoding
+title: encoding
+---
+
+# encoding
+
+The `encoding` namespace contains encoding and decoding functions.
+
+## encoding.from_base64
+
+The `encoding.from_base64` function decodes a RFC4648-compliant Base64-encoded string 
+into the original string. 
+
+`encoding.from_base64` fails if the provided string argument contains invalid Base64 data. 
+
+### Examples
+
+```
+> encoding.from_base64("dGFuZ2VyaW5l")
+tangerine
+```
+
+## encoding.from_json
+
+The `encoding.from_json` function decodes a string representing JSON into an {{< param "PRODUCT_NAME" >}} value.
+`encoding.from_json` fails if the string argument provided can't be parsed as JSON.
+
+A common use case of `encoding.from_json` is to decode the output of a [`local.file`][] component to an {{< param "PRODUCT_NAME" >}} value.
+
+> Remember to escape double quotes when passing JSON string literals to `encoding.from_json`.
+>
+> For example, the JSON value `{"key": "value"}` is properly represented by the string `"{\"key\": \"value\"}"`.
```

Comment:
```suggestion
{{< admonition type="note" >}}
Remember to escape double quotes when passing JSON string literals to `encoding.from_json`.

For example, the JSON value `{"key": "value"}` is properly represented by the string `"{\"key\": \"value\"}"`.
{{< /admonition >}}
```

We should try to avoid blockquotes and use the admonitions so that things get rendered correctly in all the various places.